priam/src/main/java/com/netflix/priam/defaultimpl/PriamConfiguration.java

```diff
@@ -207,6 +208,7 @@
     private final String DEFAULT_PARTITIONER = "org.apache.cassandra.dht.RandomPartitioner";
     public static final String DEFAULT_AUTHENTICATOR = "org.apache.cassandra.auth.AllowAllAuthenticator";
     public static final String DEFAULT_AUTHORIZER = "org.apache.cassandra.auth.AllowAllAuthorizer";
+    public static final String DEFAULT_COMMITLOG_PROPS_FILE = "/conf/commitlog_archiving.properties";
```

Comment:
The default should be prefix with onfig.getCassHome() for backward compatibility.